packages/openapi-generator/src/parser/parameters.ts

```diff
@@ -26,12 +26,25 @@ export function parseParameters(
   parameters = filterDuplicateParams(parameters);
   parameters = reorderParameters(parameters);
   parameters = renameEquallyNamedParams(parameters);
-  return parameters.map(param => ({
+  return parameters.map(param => toOpenApiParameter(param, refs));
+}
+
+function toOpenApiParameter(
```

Comment:
I think it is a good rule of thumb to name functions using a verb. I also think it is not relevant to mention OpenApi in the function name
```suggestion
function parseParameter(
```